src/com/hms_networks/americas/sc/taginfo/TagInfo.java

```diff
@@ -30,6 +30,9 @@
   /** List of tag groups */
   private final ArrayList tagGroups;
 
+  /** Integer tags can be used to represent string values in a string mapping */
+  private final String[] enumeratedStringValueMapping;
```

Comment:
Since now TagInfo contains type specific information would it make sense to break it out to

TagInfoInt
TagInfoBool
etc
then just have them inherit from TagInfo?

Seems strange to add a field here for just one special type of tag.